account_payment_term_extension/models/account_payment_term.py

```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Copyright 2013-2016 Camptocamp SA (Yannick Vaucher)
 # Copyright 2004-2016 Odoo S.A. (www.odoo.com)
 # Copyright 2015-2016 Akretion
 # (Alexis de Lattre <alexis.delattre@akretion.com>)
 # Copyright 2018 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-
+from functools import reduce
```

Comment:
Why this?

Reply:
reduce function is used here:
https://github.com/OCA/account-invoicing/pull/375/files#diff-d0188f250f37daab85f1680d28b455ccR198
and in python3 does not exist. We have two solutions, import from functools or replace reduce function by a for loop.
I choose import option.

Reply:
OK, thanks for the explanations